rocketpy/Flight.py

```diff
@@ -2345,9 +2374,7 @@ def retrieve_acceleration_arrays(self):
                 callback(self)
             # Loop through time steps in flight phase
             for step in self.solution:  # Can be optimized
-                if initTime < step[0] <= finalTime or (
```

Comment:
This was (and still is, actually), the source of the problem when running the tests.
If you see something like 'cannot make function from attribute acceleration'. This code line is causing that.